### PR TITLE
:recycle: v4 update md

### PR DESCRIPTION
**Pull request by sebastianMindee**
Add missing MD files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,15 +25,37 @@ from mindee import Client, product
...
 ```
 
+**Note:** Files can also be loaded from:
+
+A python `BinaryIO` compatible file:
+```python
+input_doc = mindee_client.source_from_file(my_file)
+```
+
+A URL (`https` only):
```
Comment (ianardee) [inline README.md:44]:
```suggestion
A URL (`HTTPS` only):
```

File: README.md [hunk 2]
```diff
@@ -25,15 +25,37 @@ from mindee import Client, product
...
+```python
+input_doc = mindee_client.source_from_url("https://my-url")
```
Comment (ianardee) [inline README.md:46]:
```suggestion
input_doc = mindee_client.source_from_url("https://example.com/path/to/file.ext")
```

File: README.md [hunk 3]
```diff
@@ -25,15 +25,37 @@ from mindee import Client, product
...
+```
+
+A base64-encoded string:
+```python
+input_doc = mindee_client.source_from_b64string(my_input_string, "my-file-name")
+```
```
Comment (ianardee) [inline README.md:52]:
```suggestion
A base64-encoded string, making sure to specify the extension the file name:
```python
input_doc = mindee_client.source_from_b64string(my_input_string, "filename.ext")
```
```

File: README.md [hunk 4]
```diff
@@ -25,15 +25,37 @@ from mindee import Client, product
...
+
+Raw bytes:
+```python
+input_doc = mindee_client.source_from_bytes(my_raw_bytes_sequence, "my-file-name")
```
Comment (ianardee) [inline README.md:56]:
```suggestion
Raw bytes, making sure to specify the extension the file name:
```python
input_doc = mindee_client.source_from_bytes(my_raw_bytes_sequence, "filename.ext")
```

File: README.md [hunk 5]
```diff
@@ -57,16 +79,18 @@ print(result.document)
 from mindee import Client, product
 
 # Init a new client and add your custom endpoint (document)
-mindee_client = Client(api_key="my-api-key").add_endpoint(
-    account_name="john",
-    endpoint_name="wnine",
+mindee_client = Client(api_key="my-api-key")
+custom_endpoint = mindee_client.create_endpoint(
+    account_name="test",
+    endpoint_name="testis",
```
Comment (ianardee) [inline README.md:85]:
```suggestion
    account_name="john",
    endpoint_name="wnine",
```

File: docs/guide/cli.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+### Example command help
+
+```shell
+python3 -m mindee parse --help
+```
+
+### Example parse command for Off-the-Shelf document
+
+```shell
+python3 -m mindee parse invoice --invoice-key xxxxxxx /path/to/invoice.pdf
```
Comment (ianardee) [inline docs/guide/cli.md:20]:
```suggestion
python3 -m mindee parse invoice --key xxxxxxx /path/to/invoice.pdf
```

**Review by ianardee — CHANGES_REQUESTED** [review]

File: README.md [hunk 7]
```diff
@@ -25,15 +25,37 @@ from mindee import Client, product
...
+```python
+input_doc = mindee_client.source_from_file(my_file)
+```
+
+A URL (`HTTPS` only):
+```python
+input_doc = mindee_client.source_from_url("https://files.readme.io/a74eaa5-c8e283b-sample_invoice.jpeg")
+```
+
+A base64-encoded string, making sure to specify the extension the file name:
```
Comment (ianardee) [inline README.md:49]:
```suggestion
A base64-encoded string, making sure to specify the extension of the file name:
```

File: README.md [hunk 8]
```diff
@@ -25,15 +25,37 @@ from mindee import Client, product
...
+```python
+input_doc = mindee_client.source_from_b64string(my_input_string, "my-file-name.ext")
+```
+
+Raw bytes, making sure to specify the extension the file name:
```
Comment (ianardee) [inline README.md:54]:
```suggestion
Raw bytes, making sure to specify the extension of the file name:
```

File: README.md [hunk 9]
```diff
@@ -57,16 +79,18 @@ print(result.document)
 from mindee import Client, product
 
 # Init a new client and add your custom endpoint (document)
-mindee_client = Client(api_key="my-api-key").add_endpoint(
-    account_name="john",
+mindee_client = Client(api_key="my-api-key")
+custom_endpoint = mindee_client.create_endpoint(
+    account_name="hohn",
```
Comment (ianardee) [inline README.md:84]:
```suggestion
    account_name="john",
```

**Review by ianardee — APPROVED** [review]

File: README.md (hunk 9 above)
Comment (sebastianMindee) [reply README.md:84]: Hohn is actually a name: https://en.wikipedia.org/wiki/Hohn

Though to be fair that was 100% a typo

**Review by ianardee — APPROVED** [review]

**Pull request merged**